docs/build_version_doc/build_all_version.sh

```diff
@@ -24,7 +24,7 @@
 tag_list="0.11.0.rc3 master"
 
 mxnet_url="https://github.com/apache/incubator-mxnet.git"
-mxnet_folder="apache_mxnet"
+mxnet_folder="mxnet"
```

Comment:
cc @piiswrong, I didn't know if this folder was purposely different from `mxnet` to avoid conflicts

Reply:
This is not part of doc

Reply:
Oh duh >.<